test/shared/src/main/scala/zio/test/environment/package.scala

```diff
@@ -681,6 +682,13 @@ package object environment extends PlatformSpecific {
       def feedLines(lines: String*): UIO[Unit] =
         consoleState.update(data => data.copy(input = lines.toList ::: data.input))
 
+      /**
+       * Returns the contents of the output buffer. The first value written to
+       * the output buffer will be the first in the sequence.
+       */
+      val errOutput: UIO[Vector[String]] =
```

Comment:
Maybe `outputErr` for IDE discoverability? Also update Scaladoc.

Reply:
Sure.

I wasn't quite sure what if you'd like to have the member in the Data class changed too, so I've left it for now. (It seems reasonable to me since it isn't really an issue for IDE discoverability and it is a "thing"... "the error output".)

Let me know if you'd like me to change that too -- I have no strong opinion on it :).